modules/cnv-restoring-node-network-configuration.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-configuring-node-network-policy
+
+[id="cnv-restoring-node-network-configuration_{context}"]
+= Restoring node network configuration after removing an interface
+
+Removing an interface from a node does not automatically restore the node network configuration to a previous state. If you remove an interface, any of the node NICs throughout the cluster that were previously configured with the interface will be in a `down` state. Restore the NICs by applying a new `NodeNetworkConfigurationPolicy` manifest to the cluster.
```

Comment:
"that were previously attached/enslaved to the interface" (e.g. bridge or bonding).